package.json

```diff
@@ -4,9 +4,23 @@
   "dependencies": {
     "@hotwired/stimulus": "^3.2.2",
     "@hotwired/turbo-rails": "^7.3.0",
-    "esbuild": "^0.19.11"
+    "@reduxjs/toolkit": "^2.0.1",
+    "axios": "^1.6.5",
+    "babel-loader": "^9.1.3",
+    "esbuild": "^0.19.11",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^9.1.0",
+    "react-router": "^6.21.2",
+    "react-router-dom": "^6.21.2"
   },
   "scripts": {
-    "build": "esbuild app/javascript/*.* --bundle --sourcemap --format=esm --outdir=app/assets/builds --public-path=/assets"
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --format=esm --outdir=app/assets/builds --public-path=/assets --loader:.js=jsx"
```

Comment:
**OPTIONAL** - Good job done so far, but when building your application, please use the already installed Webpack to bundle your project's assets (JavaScript and CSS) as this current one does not work as expected please fix this 🙏
``` 
build": "webpack --config webpack.config.js
```